app/controllers/projects/new.js

```diff
@@ -15,12 +17,65 @@ const draw = new MapboxDraw({
 });
 
 export default class NewProjectController extends Controller {
-  isDrawing = false;
+  constructor(...args) {
+    super(...args);
+    this.set('selectedLots', {
+      type: 'FeatureCollection',
+      features: [],
+    });
+  }
 
-  drawMode = null;
+  lotSelectionMode = false;
 
   @service notificationMessages;
 
+  @argument
+  projectGeometryMode = null;
+
+  @argument
+  isSelectingLots = false;
+
+  developmentSiteLayer = {
+    type: 'line',
+    paint: {
+      'line-color': 'green',
+      'line-width': 4,
+    },
+  }
```

Comment:
Just stylistic things but I would make these constants above this controller class. It helps to denote these are static objects.